jobs/build/build-sync/Jenkinsfile

```
@@ -0,0 +1,113 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
+    def commonlib = buildlib.commonlib
+
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '360'
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    commonlib.mockParam(),
+		    commonlib.ocpVersionParam('BUILD_VERSION', '4'),
```

```suggestion
		     commonlib.ocpVersionParam('BUILD_VERSION', '4'),
```
also... spaces not tabs?